prepare-videochat.sh

```diff
@@ -338,7 +347,7 @@ fi
 if lsmod | grep v4l2loopback >/dev/null 2>/dev/null; then
     # module is already loaded, do nothing
     :
-else
+elif [ $CAPTURE_STREAM = v -o $CAPTURE_STREAM = av ]; then
```

Comment:
It is because if user do not want video stream, then no need to load this kernel module.